src/data/prod/basic.lean

```diff
@@ -207,6 +207,13 @@ instance is_total_right {r : α → α → Prop} {s : β → β → Prop} [is_tr
   { exact or.inr (lex.left _ _ hji) }
 end⟩
 
+@[to_additive has_smul]
+instance has_pow {γ : Type*} [has_pow α γ] [has_pow β γ] : has_pow (α × β) γ :=
```

Comment:
In `group_theory/group_action/prod.lean`, `prod.has_pow`  and its additive version `prod.has_vadd` is defined, so maybe we should remove the `to_additive` attribute

Reply:
Did you mean `has_smul` above?

Reply:
Either way, the fix is to make sure this comes _after_ the has_smul, by moving this to that file or vice versa. I imagine we will have the same problem with the pi instance.